.github/workflows/publish.yml

```diff
@@ -0,0 +1,41 @@
+name: Upload release to PyPI
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
```

Comment:
I guess we use the 22.04 image in other actions. Might be best to harmonise to `latest`.

Reply:
good point, will do!